### PR TITLE
Feature/BRPA-756-website-and-trade-polices

### PR DESCRIPTION
**Pull request by MIzidio**
#### What is the purpose of this pull request?

Created the flow to get the trade polices name, id, and currency, and the flow to get the host

#### How to test it?

[here](https://tiktok--izidio.myvtex.com/admin/graphql-ide) and search for the tiktok-tbp

##### Trade Polices
Captura de Tela 2022-01-12 às 13 31 40

##### Host
Captura de Tela 2022-01-12 às 13 32 09

#### Types of changes

- [ ] Chore (non-breaking change which doesn't change any functionalities)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: graphql/schema.graphql [hunk 1]
```diff
@@ -1,3 +1,5 @@
 type Query {
   integrationOptions: Options
+  tradePolices: [TradePolices]
```
Comment (ConradoClark) [inline graphql/schema.graphql:3]: I suggest renaming this to `tradePolicies`:
```suggestion
  tradePolicies: [TradePolicies]
```

File: node/__tests__/getHost.test.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import type Stores from '../clients/stores'
+import { STORES_MOCK } from '../mocks/storesMock'
+import { storesService } from '../services/storesService'
+import type { AppLogger } from '../typings/appLogger'
+
+describe('SalesChannel', () => {
+  it('should return a tradePolices array', async () => {
```
Comment (ConradoClark) [inline node/__tests__/getHost.test.ts:7]: I believe this test has the wrong description 😅

**Review by anamaiabap — APPROVED** [review]

File: graphql/schema.graphql (hunk 1 above)
Comment (MIzidio) [reply graphql/schema.graphql]: Typo everywhere... Done!

File: node/__tests__/getHost.test.ts (hunk 2 above)
Comment (MIzidio) [reply node/__tests__/getHost.test.ts]: Ops... I did it again... but now I believe it has the right description

**Review by ConradoClark — APPROVED** [review]

**Pull request merged**